Sparkle/SUBinaryDeltaCreate.m

```diff
@@ -439,6 +439,16 @@ BOOL createBinaryDelta(NSString *source, NSString *destination, NSString *patchF
 
     NSString *temporaryFile = temporaryPatchFile(patchFile);
     xar_t x = xar_open([temporaryFile fileSystemRepresentation], WRITE);
+    if (!x) {
+        if (verbose) {
+            fprintf(stderr, "\n");
```

Comment:
Check a few lines above where "\nGenerating delta..." is being printed out to stderr if verbose mode is on. This is done in several other places too when an error occurs and the tool aborts so the caller can print out more info if they choose to. (Generating delta.. and other messages don't end in new line in first place because the original author thought it'd be more natural to not advance to next line until current progress was done)